lib/services/jira/jira_mapped_fields.rb

```diff
@@ -67,6 +74,19 @@ def aha_type_to_string(aha_type, aha_value)
       aha_value.to_s
     end
   end
+
+  def aha_type_to_user(aha_type, field)
+
+    if aha_type == "string"
+      user_resource.picker(field.value.strip).try(:[], :key)
+    else
+      field.email_value.each do |email|
+        potential_key = user_resource.picker(email).try(:[], :key)
+        return potential_key  unless potential_key.nil?
+      end
+      nil
+    end
+  end
```

Comment:
I think the trade off is worth it here since the new way seems more bulletproof to me and easier to read and understand at a glance. The patterns in the other `aha_type_to_*` methods aren't so much alike where I think having more similar looking code is worth it. In other words, the benefits seem to me to outweigh the slight departure in coding style.